### PR TITLE
Added gen.md file.

### PR DESCRIPTION
**Pull request by Aditya-A-Thakkar**

### CODE REVIEW

File: docs/gen.md [hunk 1]
```diff
@@ -0,0 +1,297 @@
+# APBS
+
+- apbs_gen
+    - Interactive APBS input file generator. It runs the TnputApp.
+        - Args:
+            - input_file_path: The path to the input file
```
Comment (AnirudhG07) [inline docs/gen.md]: Add `## Options` before mentioning all of this

**Review by AnirudhG07 — CHANGES_REQUESTED** [review]
You dont write docs this way!
You just mentioned the options and about it. Which is also needed, but you put PLAIN english as well.
If you can add how to run the interactive apbs and what each key binding too, that would be great

File: docs/gen.md (hunk 1 above)
Comment (AnirudhG07) [reply docs/gen.md]: > If you can add how to run the interactive apbs and what each key binding too, that would be great

I mean say use `bept gen -a protein.in`, how to use, etc. Any limitations like can you provide `/absolute/path/to/input_file` or it should be present in your directory, etc.

**Review by Aditya-A-Thakkar — COMMENTED** [review]
Done the changes as requested.

**Comment by AnirudhG07** [comment]
Ill change some stuff myself. It looks better. Thanks!

**Review by AnirudhG07 — APPROVED** [review]

**Pull request merged**